marxan.cpp

```diff
@@ -374,6 +380,12 @@ namespace marxan {
             }
         }
 
+        if(quitting_loop)
+        {
+            displayProgress1("\nRuns were aborted due to error.\n"); 

```

Comment:
please insert cin.get() as the customer wants the screen to pause if error

Reply:
or insert cin.get() where appropriate (maybe not here, since the exception gets caught above)